python/doc/examples/functional_modeling/logistic_growth_model.ipynb

```diff
@@ -0,0 +1,305 @@
+    "\n",
+    "[2] Cleve Moler. Numerical Computing with Matlab. Society for Industrial Applied Mathematics,\n",
+    "2004.\n",
+    "\n",
+    "[3] Raymond Pearl and Lowell Reed. On the rate of growth of the population of the united states\n",
+    "since 1790 and its mathematical representation. Proceedings of the National Academy of Sciences,\n",
+    "1920."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define the model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function\n",
+    "import openturns as ot\n",
+    "from numpy import linspace, exp, maximum"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Some data\n",
+    "ustime=list(range(1790,2001,10))\n",
+    "uspop=[3.9,5.3,7.2,9.6,13.,17.,23.,31.,39., 50.,62.,76.,92.,106.,123.,132.,151.,179., 203.,221.,250.,281.]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Creation of the input distribution\n",
+    "y0 = 3.9e6\n",
+    "a = 0.03134\n",
+    "b = 1.5887e-10\n",
+    "distY0 = ot.Normal(y0, 0.1 * y0)\n",
+    "distA  = ot.Normal(a, 0.3 * a)\n",
+    "distB  = ot.Normal(b, 0.3 * b)\n",
+    "distX = ot.ComposedDistribution([distY0, distA, distB])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The model\n",
+    "class Popu(ot.OpenTURNSPythonPointToFieldFunction):\n",
+    "\n",
+    "    def __init__(self, t0 = 1790.0, tfinal = 2000.0, nt = 1000):\n",
+    "        grid = ot.RegularGrid(t0, (tfinal - t0) / (nt - 1), nt)\n",
+    "        super(Popu, self).__init__(3, grid, 1)\n",
+    "        self.setInputDescription(['y0', 'a', 'b'])\n",
+    "        self.setOutputDescription(['N'])\n",
+    "        self.ticks_ = [t[0] for t in grid.getVertices()]\n",
+    "        self.phi_ = ot.SymbolicFunction(['t', 'y', 'a', 'b'], ['a*y - b*y^2'])\n",
+    "        \n",
+    "    def _exec(self, X):\n",
+    "        y0 = X[0]\n",
+    "        a  = X[1]\n",
+    "        b  = X[2]\n",
+    "        phi_ab = ot.ParametricFunction(self.phi_, [2, 3], [a, b])\n",
+    "        phi_t = ot.ParametricFunction(phi_ab, [0], [0.0])\n",
+    "        solver = ot.RungeKutta(phi_t)\n",
+    "        initialState = [y0]\n",
+    "        values = solver.solve(initialState, self.ticks_)\n",
+    "        return values * [1.0e-6]\n",
+    "\n",
+    "F = Popu(1790.0, 2000.0, 1000)\n",
+    "popu = ot.PointToFieldFunction(F)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generate a sample from the model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Sample the model\n",
```

Comment:
```suggestion
    "# Sample from the model\n",
```

Reply:
Ok, integrated. I even created a specific Text cell, instead of a comment in the code. Fixed.